cbits/cbits.c

```diff
@@ -0,0 +1,56 @@
+#include <stdint.h>
+#include <string.h>
+#include <limits.h>
+#include <assert.h>
+
+/* Taken from:
+ *
+ * <http://stackoverflow.com/questions/776508/best-practices-for-circular-shift-rotate-operations-in-c>
+ *
+ * Intel is little-endian, we have to rotate right
```

Comment:
I merged this already but only now I spotted this change.  Endianness has _nothing to do with rotations_.  Shifts are defined on the numerical values, it doesn't matter how these are stored underneath.